src/GameWindow.py

```diff
@@ -312,130 +312,192 @@ def create_buttons(self, buttons_on_the_left):
         # Empty ButtonGroup
         self.buttons.removal_all()
 
+        # Dictionary of 'vector' arrays for arrow shapes
+        arrows = {
```

Comment:
Could these maybe go as an `Enum` class in Buttons.py? Similar to the `Heading` class in BeeBot.py

I'm hesitant to add more lines to GameWindow as it's already too big and too complicated.